sdk/communication/Azure.Communication.Chat/src/ChatClient.cs

```diff
@@ -281,9 +281,9 @@ public virtual Response DeleteChatThread(string threadId, CancellationToken canc
 
         private static HttpPipeline CreatePipelineFromOptions(ChatClientOptions options, CommunicationUserCredential communicationUserCredential)
         {
-            var httpPipelinePolicy = new CommunicationUserAuthenticationPolicy(communicationUserCredential);
-            HttpPipeline httpPipeline = HttpPipelineBuilder.Build(options, httpPipelinePolicy);
-            return httpPipeline;
+            var tokenCredential = new CommunicationTokenCredential(communicationUserCredential);
+            var authenticationPolicy = new BearerTokenAuthenticationPolicy(tokenCredential, "");
```

Comment:
scope : chat 
new BearerTokenAuthenticationPolicy(tokenCredential, "chat"); ?

Reply:
Spoke to @RezaJooyandeh about the scope. And since previously we were not using anything in the CommunicationUserAuthenticationPolicy it might be beneficial to just leave it blank to be consistent

Reply:
@juancamilor This scope is not sent to the identity service to issue token, that is done separately.